collector/lib/CollectorConfig.cpp

```diff
@@ -160,6 +163,48 @@ CollectorConfig::CollectorConfig(CollectorArgs* args) {
   HandleAfterglowEnvVars();
 
   host_config_ = ProcessHostHeuristics(*this);
+
+  const char* envvar;
+
+  connection_stats_quantiles_ = {0.50, 0.90, 0.95};
+
+  if ((envvar = std::getenv("ROX_COLLECTOR_CONNECTION_STATS_QUANTILES")) != NULL) {
+    connection_stats_quantiles_.clear();
+    std::stringstream quantiles(envvar);
+    while (quantiles.good()) {
+      std::string quantile;
+      std::getline(quantiles, quantile, ',');
+      try {
+        double v = std::stod(quantile);
+        connection_stats_quantiles_.push_back(v);
+        CLOG(INFO) << "Connection statistics quantile: " << v;
+      } catch (...) {
```

Comment:
[nit] Consider catching more specific errors.

Reply:
I wanted to keep the error-handling codepath to a minimum, since the INFO log gives the values that have been parsed, I think that a malformed value should be rather easy to spot.